### PR TITLE
fix: searchbar showing incorrect spacing at medium size

### PR DESCRIPTION
**Pull request by natew**
## Description

Fixes searchbar height issue

_Linear ticket:_ [WEB-2100](https://linear.app/uniswap/issue/WEB-2100/fix-global-search-results-ui-with-thin-screen-width) 

## Screen capture

Before (dark mode but irrelevant):

CleanShot 2023-05-23 at 12 48 25@2x

After:

CleanShot 2023-05-23 at 12 48 40@2x

### Reproducing the error

On the main searchbar

1. Have to get the size to be above md but below xl where the search icon is small and to the right side, ~1075px wide works

### QA (ie manual testing)

- [ ] Resize browser to 1075px wide
- [ ] Open search
- [ ] Should have a normal display matching the wider width (above 1100px)

#### Devices

All web browsers

### Automated testing

- [ ] Unit test
- [ ] Integration/E2E test

[WEB-2100]: https://uniswaplabs.atlassian.net/browse/WEB-2100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cmcewen — APPROVED** [review]

**Review by tinaszheng — APPROVED** [review]

**Comment by natew** [comment]
@cmcewen thank you, after [some discussion](https://uniswapteam.slack.com/archives/C03NMB3PCL8/p1684949590305809) thinking [this is a better fix](https://github.com/Uniswap/interface/pull/6637/commits/597bdb34ba19011b041f56d0a6d68e68d8b7ce65):

File: .vscode/settings.json [hunk 1]
```diff
@@ -12,5 +12,14 @@
...
+  "eslint.debug": true,
+  "[javascript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
```
Comment (zzmp) [inline .vscode/settings.json:24]: Could you put this in a separate PR, so it's not grouped with the unrelated fix?

Comment (natew) [reply .vscode/settings.json]: All done

File: src/components/Web3Status/index.tsx [hunk 2]
```diff
@@ -107,7 +108,7 @@ const Web3StatusConnected = styled(Web3StatusGeneric)<{
 const AddressAndChevronContainer = styled.div`
   display: flex;
 
-  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.navSearchInputVisible}px`}) {
+  @media only screen and (max-width: ${() => `${navSearchInputVisibleSize}px`}) {
```
Comment (cmcewen) [inline src/components/Web3Status/index.tsx:111]: this can just be static now and not a function right?

**Comment by infredible** [comment]
A couple issues I'm seeing at the 1100px breakpoint
1. The implementation has the search box off to the right. It should be centered and appear "floating" over the UI
2. Add a scrim behind the search window (only at this breakpoint)
3. Add an X button on the right in the input field (clicking this or on the scrim should dismiss)

**Current**
CleanShot 2023-05-25 at 10 13 30@2x

**Desired**
image

Figma: https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-Design-Spec?type=design&node-id=5772%3A126780&t=w6ddEFGRO2q1lO7P-1

**Comment by natew** [comment]
@infredible looks good, mind if I continue that on a follow on pr as this one fixes the bug and cleans up some code that is ready to go?

File: src/components/Web3Status/index.tsx (hunk 2 above)
Comment (natew) [reply src/components/Web3Status/index.tsx]: Good catch, going to take a bit to get used to styled-components / vanilla extract.

**Comment by natew** [comment]
@infredible my uniswap figma doesn't seem to have access to that file, maybe I need to finish some sort of onboarding steps or get access to a team?

**Comment by natew** [comment]
@cmcewen @tinaszheng should be addressed now!

**Review by cmcewen — APPROVED** [review]
:shipit:

**Review by cmcewen — APPROVED** [review]
:shipit:
